entity-framework/core/miscellaneous/cli/dotnet.md

```diff
@@ -174,28 +244,36 @@ Removes the last migration.
 
 Options:
 
-|    |         |                                                                       |
-|:---|:--------|:----------------------------------------------------------------------|
-| -f | `--force` | Revert the migration if it has been applied to the database. |
+|                   | Option    | Description                                                             |
+|-------------------|-----------|-------------------------------------------------------------------------|
+| <nobr>`-f`</nobr> | `--force` | Revert the migration even if it has been applied to the database.       |
```

Comment:
Not sure *even* makes sense here. Reverting and removing are different:

* Reverting--rolls back the changes in the migration from the database
* Removing--deletes the migration code from the project